docs/src/main/asciidoc/kafka/kafka_partitions.adoc

```diff
@@ -54,7 +54,13 @@ spring:
             partition-count: 12
 ----
 
-IMPORTANT: The topic must be provisioned to have enough partitions to achieve the desired concurrency for all consumer groups.
+IMPORTANT: It is important to keep in mind that since Apache Kafka supports partitioning natively, there is no need to rely on binder partitioning as described above unless you are using custom partition keys as in the example or an expression that involves the payload itself.
```

Comment:
```suggestion
IMPORTANT: It is important to keep in mind that, since Apache Kafka supports partitioning natively, there is no need to rely on binder partitioning as described above unless you are using custom partition keys as in the example or an expression that involves the payload itself.
```